build.sbt

```diff
@@ -282,6 +279,7 @@ lazy val scalameta = crossProject(allPlatforms: _*).in(file("scalameta/scalameta
 lazy val semanticdbIntegration = project.in(file("semanticdb/integration")).settings(
   description := "Sources to compile to build SemanticDB for tests.",
   sharedSettings,
+  crossScalaVersions := AllScalaVersions,
```

Comment:
is this just for testing?

Reply:
Yep, it's only used for testing and should really be AllScalaVersions as tests are also AllScalaVersions. I had some issues otherwise with Scala versions and this should be fine.